### PR TITLE
updated docs to remove note about windows tests not working

### PR DESCRIPTION
**Pull request by mwarger**
`flutter test` is now supported on Windows, so I removed the line about it not being supported.  The formatting has also been cleaned up (thanks to prettier).

### CODE REVIEW

**Comment by mwarger** [comment]
I signed it!

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,63 +1,59 @@
-Contributing to Flutter
-=======================
+# Contributing to Flutter
```
Comment (Hixie) [inline CONTRIBUTING.md]: I'm not sure this is prettier... if someone is reading this in their text editor (which seems at least as likely as reading it on the Web site) it's a lot clearer that something is a header if it's underlined.

**Comment by Hixie** [comment]
LGTM

**Comment by Hixie** [comment]
Thanks for your patch, I'll land it when the tree is green.

**Pull request merged**